src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const pickedImg = document.getElementById('thumbs');
+
+pickedImg.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (e.target.tagName === 'IMG' && e.target.parentElement.tagName === 'A') {
+    const linkElement = e.target.parentElement;
+    const imageUrl = new URL(
+      linkElement.getAttribute('href'),
+      window.location.href,
+    );
+
+    largeImg.setAttribute('src', imageUrl);
+  }
+});
```

Comment:
You can simplify your code like this 
```suggestion
const largeImg = document.getElementById('largeImg');
const gallery = document.querySelector('.gallery');

gallery.addEventListener('click', (e) => {
  e.preventDefault();

  largeImg.src = e.target.closest('.list-item__link').href;
});
```